esp/esp/program/modules/forms/splashinfo.py

```diff
@@ -33,32 +33,25 @@
 """
 
 from django import forms
-from esp.middleware import ESPError
-from esp.tagdict.models import Tag
 import simplejson as json
 
-"""
-The SplashInfoForm is customizable for different lunch options.
-The choices should be provided in a Tag, splashinfo_choices, which is in
-JSON format.  Example:
-    key = 'splashinfo_choices'
-    value = '{
-        "lunchsat": [["no", "No thanks; I will bring my own lunch"],
-                  ["chicken", "Yes, Chicken"],
-                  ["steak", "Yes, Steak"],
-                  ["spicy_thai", "Yes, Spicy Thai"],
-                  ["veggie", "Yes, Vegetarian"]],
-        "lunchsun": [["no", "No thanks; I will bring my own lunch"],
-                  ["cheese", "Yes, Cheese Pizza"],
-                  ["pepperoni", "Yes, Pepperoni Pizza"]],
-    }'
-
-The interface (including directions, whether to exclude or rename days,
-and costs) should be controlled by overriding the template:
-    program/modules/splashinfomodule/splashinfo.html
-"""
+from esp.accounting.models import LineItemType
+from esp.accounting.controllers import IndividualAccountingController
+
+
+LUNCH_SATURDAY = 'lunchsat'
+LUNCH_SUNDAY = 'lunchsun'
+DISCOUNT_SIBLING = 'siblingdiscount'
+SIBLING_NAME = 'siblingname'
```

Comment:
The values were *previously* hardcoded - I created the variables for reuse and so this doesn't change a whole lot, materially, except that it's probably better to reference variables, rather than string literals. In terms of making the values dynamic, this opens up the discussion a bit around the feature requirements. i.e. If we want to provide more flexibility beyond the weekend lunch and sibling discount - what would that be?(and is this redundant, when considering that it could be handled by studentreg?)